File: model/user_playlists.go
package model

import (
	"github.com/jinzhu/gorm"
)

type UserPlaylists struct {
	gorm.Model
	UserID     int64 `gorm: "column: user_id; type: bigint; primary_key; auto_increment: false; unique; not null"`
	PlaylistID int64 `gorm: "column: playlist_id type: bigint; primary_key; auto_increment: false; unique; not null"`
}

File: admin/backend/src/db/model/user-playlists.ts
import { Model, snakeCaseMappers } from 'objection';

export class UserPlaylists extends Model {
    userId: string;
    playlistId: string;
    
    static get tableName() {
        return 'user_playlists';
    }

    static get idColumn() {
        return ['user_id', 'playlist_id'];
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default UserPlaylists;